osu.Game.Tests/Visual/Navigation/TestSceneChangeAndUseGameplayBindings.cs

```diff
@@ -50,8 +50,11 @@ public void TestGameplayKeyBindings()
             AddStep("close settings", () => Game.Settings.Hide());
 
             AddStep("import beatmap", () => BeatmapImportHelper.LoadQuickOszIntoOsu(Game).WaitSafely());
+
             PushAndConfirm(() => new PlaySongSelect());
 
+            AddUntilStep("wait for selection", () => !Game.Beatmap.IsDefault);
+
```

Comment:
This seems to have still failed the recent CI run: https://github.com/ppy/osu/runs/4934221900?check_suite_focus=true

Reply:
Probably going to be a zero volume notification or similar...

```csharp

322
 [runtime] 2022-01-25 09:16:14 [debug]: Focus on "NotificationOverlay" no longer valid as a result of unfocusIfNoLongerValid.
323
 [runtime] 2022-01-25 09:16:14 [debug]: Focus changed from NotificationOverlay to nothing.
324
 [runtime] 2022-01-25 09:16:14 [debug]: Focus changed from nothing to NotificationOverlay.
```